### PR TITLE
schemas: phd_advisors -> advisors

### PR DESCRIPTION
**Pull request by jacquerie**
Sentry: https://sentry.cern.ch/inspire-sentry/inspire-labs-qa/group/601538/

### CODE REVIEW

**Comment by jacquerie** [comment]
Ready for review.

**Comment by mihaibivol** [comment]
LGTM :D

File: inspirehep/dojson/hepnames/fields/bd1xx.py [hunk 1]
```diff
@@ -393,36 +393,37 @@ def experiments2marc(self, key, value):
...
-        )
+
+    _degree_type = force_single_element(value.get('g'))
+    degree_type = DEGREE_TYPES_MAP.get(_degree_type, 'Other')
+
     return {
-        'id': value.get("i"),
-        'name': value.get("a"),
-        'degree_type': degree_type
+        'id': value.get('i'),
```
Comment (jacquerie) [inline inspirehep/dojson/hepnames/fields/bd1xx.py]: Not part of the schema. This actually needs the terrible identifier problem to be solved.

**Comment by jacquerie** [comment]
Just removed the `id` field which was wrongly being populated, and added a FIXME comment about `701__i`.

**Pull request merged**